app/helpers/users_helper.rb

```diff
@@ -0,0 +1,6 @@
+module UsersHelper
+  def user_age(user)
+    now = Time.now.utc.to_date
+    now.year - user.birthday.year - ((now.month > user.birthday.month || (now.month == user.birthday.month && now.day >= user.birthday.day)) ? 0 : 1)
```

Comment:
That line has a lot of logic in it and it's really hard to follow it.
It would be nice to make it more readable - try splitting it to multiple lines, assigning parts of it into well-named variables etc., please 😄